pages/admin/dashboard.js

```diff
@@ -0,0 +1,229 @@
+    if (state === "END") {
+      setGameClick(false);
+      setGamePin("");
+    }
+    firebase
+      .database()
+      .ref(`games/` + gamepin)
+      .child("state")
+      .set("GAME_STATE-" + state);
+  }
+
+  function signOutUser() {
+    firebase
+      .auth()
+      .signOut()
+      .then(() => {
+        // Sign-out successful.
+        setLogin(false);
+      })
+      .catch((error) => {});
+  }
+
+  return (
+    <>
+      {gameClick ? (
+        <div>
+          <Link href="dashboard">
+            <h1 onClick={dashMain}>Back to dashboard</h1>
+          </Link>
+          <Container id={styles.listContainer} style={{ minHeight: "10vh" }}>
+            <Typography variant="h4" gutterBottom>
+              Pin: {gamePin}
+            </Typography>
+            {console.log()}
+            <div style={{ display: "flex", flexDirection: "row" }}>
+              <Link href={"/admin/game?gamepin=" + gamePin}>
+                <Button
+                  variant="contained"
+                  onClick={() => {
+                    gameArrayState[gamePin].state === "GAME_STATE-WAITING" &&
+                      stateToggle(gamePin, "GAME_QUESTION_1");
+                  }}
+                  color="primary"
+                  id={styles.enterButton}
+                >
+                  {gameArrayState[gamePin].state === "GAME_STATE-WAITING"
+                    ? "Start Game"
+                    : "View Game"}
+                </Button>
+              </Link>
+
+              <Link href="dashboard">
+                <Button
+                  variant="contained"
+                  onClick={() => stateToggle(gamePin, "END")}
+                  color="primary"
+                  id={styles.enterButton}
+                >
+                  End game
+                </Button>
+              </Link>
+            </div>
+
+            <Typography variant="h5" style={{ marginTop: "20px" }} gutterBottom>
+              Game state:{" "}
+              {gameArrayState[gamePin].state.replace("GAME_STATE-", "")}
+            </Typography>
+
+            <Card>
+              <CardContent id={styles.nameContainer}>
+                {gameArrayState[gamePin].users !== undefined &&
+                  Object.keys(gameArrayState[gamePin].users).map(function (
+                    key
+                  ) {
+                    return (
+                      <div style={{ display: "flex", flexDirection: "row" }}>
+                        <div
+                          onClick={() => shadowToggle(gamePin, key)}
+                          style={{
+                            textDecoration:
+                              gameArrayState[gamePin].users[key].sban !==
+                                undefined &&
+                              gameArrayState[gamePin].users[key].sban
+                                ? "line-through"
+                                : "",
+                          }}
+                          id={styles.username}
```

Comment:
```suggestion
                          className={styles.username}
```
and change the `#username` in the CSS file to `.username`.

this prevents multiple elements having the same id